### PR TITLE
Permit requesting transcriptions in text/plain

### PR DESCRIPTION
**Pull request by benlangfeld**

### CODE REVIEW

**Comment by benlangfeld** [comment]
This replaces #15

File: test/test_helper.rb [hunk 1]
```diff
@@ -8,6 +8,8 @@
 require 'minitest/autorun'
 require 'minitest/spec'
 
+require 'byebug'
```
Comment (pkarman) [inline test/test_helper.rb]: is this necessary? or just left behind from debugging?

Comment (benlangfeld) [reply test/test_helper.rb]: This allows one to insert ‘byebug’ in specs when writing them to debug. Without this, what’s the point in specifying the dependency? It’s there to make one’s life easy when working on specs, no?

Comment (pkarman) [reply test/test_helper.rb]: Ah I didn't realize it was already in the gemspec as a dependency. Thanks!

**Pull request merged**